test/terraform/modules/cluster/gke/input/gke-nongpu.tfvars

```diff
@@ -0,0 +1,11 @@
+project_id      = "gce-ai-infra"
+resource_prefix = "ci"
+region          = "us-central1"
+node_pools = [{
+  zone                     = "us-central1-a"
+  node_count               = 1
+  machine_type             = "n1-standard-1"
+  guest_accelerator_type   = ""
+  guest_accelerator_count  = 0
```

Comment:
Do you think this should be preferred over `guest_accelerator { type = string, count = number}` (and here `guest_accelerator = null`)?

Reply:
addressed in latest iteration.